charts/pega/values-minimal.yaml

```diff
@@ -83,6 +83,9 @@ global:
       url: "YOUR_DOCKER_REGISTRY"
       username: "YOUR_DOCKER_REGISTRY_USERNAME"
       password: "YOUR_DOCKER_REGISTRY_PASSWORD"
+    # To avoid exposing docker registry details, create secrets for docker registry credentials.
```

Comment:
To avoid exposing Docker registry details, create secrets to manage your Docker registry credentials.